### PR TITLE
Task5261 ui audio

### PR DESCRIPTION
**Pull request by NguyenQuocBao19182**
https://edu-redmine.framgia.vn/issues/5261
screenshot from 2019-01-15 13-12-34

### CODE REVIEW

File: app/src/main/res/layout/fragment_audio.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layoutManager="android.support.v7.widget.LinearLayoutManager"
+        app:layout_constraintTop_toBottomOf="@+id/textViewAudioGenre"
+        />
+    <TextView
+        android:id="@+id/textViewListMusicAudio"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="List Music"
```
Comment (buivanhieu7112) [inline app/src/main/res/layout/fragment_audio.xml]: hard code

File: app/src/main/res/layout/fragment_audio.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/bg_color"
+    >
+    <TextView
+        android:id="@+id/textViewAudioGenre"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="Genres"
```
Comment (buivanhieu7112) [inline app/src/main/res/layout/fragment_audio.xml]: hard code

File: app/src/main/res/layout/item_view_list_music.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        android:layout_marginTop="@dimen/dp_5"
+        android:textColor="@color/white"
+        android:id="@+id/textViewNameArticsItemListMusic"
+        android:layout_width="@dimen/dp_300"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="@dimen/dp_5"
+        app:layout_constraintLeft_toRightOf="@+id/imageViewItemListMusic"
+        app:layout_constraintTop_toBottomOf="@id/textViewSongNameItemList"
+        />
+</android.support.constraint.ConstraintLayout>
```
Comment (buivanhieu7112) [inline app/src/main/res/layout/item_view_list_music.xml:36]: reformat code

**Review by buivanhieu7112 — APPROVED** [review]

**Comment by NguyenQuocBao19182** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**